seqr/utils/middleware.py

```diff
@@ -20,6 +21,7 @@
 EXCEPTION_ERROR_MAP = {
     PermissionDenied: 403,
     ObjectDoesNotExist: 404,
+    Http404: 404,
```

Comment:
Are there any specific cases that raise ObjectDoesNotExist or Http404 exception? I can't find any in seqr.

Reply:
these are django builtin errors, so while we never reaise them ourselves they happen all the time in the django library code. `Http404` happens for example whenever we request an image file path that doesn't exist, which is the issue that this PR is fixing, and ObjectDoesNotExist is raised if you do a get query for something that doesn't exist